LoRaEngine/LoraKeysManagerFacade/CreateEdgeDevice.cs

```diff
@@ -71,7 +99,7 @@ public static async System.Threading.Tasks.Task<HttpResponseMessage> RunAsync([H
                 await manager.AddDeviceAsync(endDevice);
                 Twin endTwin = new Twin();
                 endTwin.Tags = new TwinCollection(@"{AppEUI:'BE7A0000000014E2',AppKey:'8AFE71A145B253E49C3031AD068277A1',GatewayID:''," +
-                "SensorDecoder:'DecoderRotatorySensor'}");
+                "SensorDecoder:'DecoderValueSensor'}");
```

Comment:
Do you know what the `deployEndDevice`  with the hard coded AppEUI and AppKey is used for? Sounds like  test functionality.

Reply:
If you use the template and you set the "DeployDevice" option to true. It will also deploy a demo end device to the iot Hub that will be immediately compatible with the Arduino code we have. To this end we need to hard set APPEUI and APPKEY. As the aim of this is educational only I think it's fine.

Reply:
You should change the SensorDecoder to 'DecoderValueSensor' on the comment of the Arduino OTAA example to the new one. I changed mine already on the ABP example